Runtime/EcsactDefaults.cs

```diff
@@ -0,0 +1,35 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("EcsactRuntimeDefaults")]
+
+namespace Ecsact {
+	static public class Defaults {
+		public static EcsactRuntime? Runtime;
+		// NOTE(KELWAN) Default registry is currently guaranteed
+		public static Ecsact.Registry Registry;
+		public static Ecsact.UnitySync.EntityGameObjectPool? Pool;
+		public static EcsactRunner Runner;
+
+		private static global::System.Action onReady;
```

Comment:
I think you meant to make this an event?
```suggestion
		private static event global::System.Action onReady;
```